### PR TITLE
[13.0][MIG] resource_hook

### PR DESCRIPTION
**Pull request by jarroyomorales**
Migration to v13 of `resource_hook`

@etobella @Saran440 @Abranes @Jortolsa @AaronHForgeFlow

### CODE REVIEW

**Review by etobella — APPROVED** [review]
Tested with #897 And worked fine

File: resource_hook/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+12.0.1.0.2 (2020-06-09)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Introduce hook in method `get_work_hours_count` of ResourceCalendar class in
+  order to allow changes in the logic to compute working hours within an
+  interval. A new method `_get_work_hours` is now provided for this purpose.
+  The method can be inherit to alter the standard behaviour.
+
+
+13.0.1.0.1 (2020-09-29)
```
Comment (Saran440) [inline resource_hook/readme/HISTORY.rst]:
```suggestion
13.0.1.0.0 (2020-09-29)
```

**Review by Saran440 — APPROVED** [review]
Functional Test :+1: 

Code Reviewed.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Looks good!

**Pull request merged**